kafka-connect-aws-s3/src/main/scala/io/lenses/streamreactor/connect/aws/s3/sink/S3WriterManager.scala

```diff
@@ -102,19 +108,28 @@ class S3WriterManager(sinkName: String,
   }
 
   def write(topicPartitionOffset: TopicPartitionOffset, messageDetail: MessageDetail): Unit = {
-    logger.debug(s"[$sinkName] Received call to S3WriterManager.write for ${topicPartitionOffset.topic}-${topicPartitionOffset.partition}:${topicPartitionOffset.offset}")
+    try {
```

Comment:
`handleTry`  should basically be wrapping the `write` function:


```
 def write(topicPartitionOffset: TopicPartitionOffset, messageDetail: MessageDetail): Unit = {
   handleTry{
     Try{
       <function body>
     }
  }
}
```

So you don't need to put the try-catch yourself